### DISCUSSION TITLE
I can't verify my student status.

### DISCUSSION THREAD

**Discussion by pinetree2** [community/community#51221] [GitHub Education]
Even though I have my school name, my major, and my name in my profile, I am not verified as a student. I have provided a leave of absence certificate, but I am not sure what the problem is.

**Comment by Ntshangase** [comment]
Hi, 

Please provide more information like a screenshot of rejection reason and tell us about the type of documents you submitted.

Tip: make sure that your location is not blocked in your browser and GitHub does use location during the application process.

**Reply by Ntshangase** [reply]
Hello @pinetree2  

Did you manage to get your problem solved?

**Reply by pinetree2** [reply]
@Ntshangase 

<img width="458" alt="image" src="https://github.com/community/community/assets/79689822/c52a8c39-60f7-4d32-a45c-4e2b8116e542">
I just checked the notification.
I wrote all the information on my profile, but I don't recognize my student card.
The reason why I used my student ID is that the certificate of leave of absence mentioned in the article was not certified.

I am attaching my native language as I think the intention will be different using a translator.

알림을 이제야 확인했습니다.
안내받은대로 프로필의 모든 설정을 학생증과 같이 설정하였음에도 불구하고, 인증이 5~6번이나 되지 않습니다.
본문에서 언급한 휴학 증명서로는 되지않았기 때문에 학생증으로 시도한 것입니다.

문제를 해결할 방법을 주시면 
정말 감사하겠습니다.!!

**Reply by Ntshangase** [reply]
ok, For the document that you provided you need to make sure that you also provide an English translation for document. 
For doing translations check out: [Translations](https://www.luogu.com.cn/blog/pnpn111/github-student-packet)

- Based on this discussion: [Similar discussion](https://github.com/orgs/community/discussions/52728)

**Comment by sumitshinde-84** [comment]
1.I have occasionally changed my browser and when I process my application, there was a message trying to get my location (never seen before). It may be important to keep the location the same as my school.
2.Using the dated student ID card, as @KleeMatrix mentioned above. I have tried with other document, and they just didn't work.


@pinetree2 try this because same issue i was facing and now i got verified so i have followed above steps

**Comment by samus-aran** [comment]
Hi @pinetree2 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by pinetree2** [reply]
@samus-aran 

![image](https://github.com/community/community/assets/79689822/f9c5164c-4ff2-4b6e-8b3a-18a5328f79ba)
I just checked the notification. I wrote all the information on my profile, but I don't recognize my student card. The reason why I used my student ID is that the certificate of leave of absence mentioned in the article was not certified.
I am attaching my native language as I think the intention will be different using a translator.

알림을 이제야 확인했습니다.
안내받은대로 프로필의 모든 설정을 학생증과 같이 설정하였음에도 불구하고, 인증이 5~6번이나 되지 않습니다.
본문에서 언급한 휴학 증명서로는 되지않았기 때문에 학생증으로 시도한 것입니다.

문제를 해결할 방법을 주시면
정말 감사하겠습니다.!!

**Comment by Domwis-IR** [comment]
I also want to know whether a leave of absence certificate could work as a proof of student. A leave of absence certificate is that I am now resting my academic year but I am still student and not graduated yet. Is this file still can work as a proof that I am student?
